### ISSUE TITLE
[v1] QSelect map-options displays value if no match found

### ISSUE THREAD

**Issue by jean-moldovan** [quasarframework/quasar#4288]
Hello!

Wanted to raise this case, even though this is not really a bug.

Scenario:
1) You have QSelect and use emit-value and map-options
2) Options are loaded async
3) Select is visible while options are loading, spinner is displayed

https://codepen.io/anon/pen/KLJeMK?&editable=true&editors=101

Problem:
Until options are loaded select displays values itself, and not the label. Once options are ready, it displays the label correctly.

Is this intentional, or maybe it would make more sense to have empty display label if no match found with `map-options`?

Thanks!

**Comment by pdanpdan** [comment]
Check here for how to workaround this problem: https://github.com/quasarframework/quasar/issues/4250

**Comment by jean-moldovan** [comment]
Okay, thanks! Feel free to close if you think that this is not something that should be fixed in the scope of Quasar.

**Comment by jean-moldovan** [comment]
My proposal was to display empty label if no match found using `map-options`.

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.5.5

**Comment by ryhn** [comment]
Hi, 
Is this issue fixed ? I am using quasar 1.15.22 In my case, model value is not in options but if model has a value, it is appear on qselect in span html tag. According to me Qselect behaviour should be default select element of html. Value should be in option list.

My Qselect code;
```
<q-select
     v-model="frm.store_id.value"
     :options="storeList"
      name="store_id"
      label="Store *"
      map-options
      emit-value
 />
```
